src/services/loginService.ts

```diff
@@ -7,6 +7,14 @@ import {
   InteractiveLoginOptions,  
 } from "@azure/ms-rest-nodeauth";
 
+export interface AzureLoginOptions {
```

Comment:
You are only using `subscriptionId`.  I would remove the other options and let @PIC123 add these when he adds his PR.